_usage/limitations-of-babelfish.md

```diff
@@ -9,89 +9,90 @@ nav_order: 5
 In this chapter, you will learn about Babelfish limitations and compatibility.
 There is never going to be 100% compatibility between large database systems.
 However, much can be done to stay as compatible as possible, which is definitely
-true for Babelfish.
+the goal of Babelfish.
 
 Let's dive in and figure out what works, and where challenges still lay ahead.
 
 ### Missing features
 
-The following table contains a list of missing features. Note that the list of
-limitations will change in the future as more features are added, new missing
-features are discovered, and additional features are added to MS SQL:
+The following table contains a list of SQL Server features that are currently
+not implemented in Babelfish.  Note that the list of limitations will change
+in the future as more features are added and additional features are added to
+Microsoft SQL Server.
```

Comment:
Dont comment on competitor products. We are not really planning to keep track of the new features in future MSSQL versions. Instead, say that the list with change because some of these curently unsupported features will be supported by BBF.

Reply:
I'll leave it as it is, but change "will change" to "may change".
I am not commenting on competitor's products, this is about Babelfish.
Even if Amazon doesn't track changes in SQL Server, I see no reason why a new SQL Server feature could make it into this list.

Reply:
We should simply not mention what may happen to SQL Server in the future. We don't care.

Reply:
Well, then we agree to disagree.

Reply:
I have remofed the reference to Microsoft in e325eecac25fb3d660777fd242ae8bb6eea07db2.